cosmicds/app.vue

```diff
@@ -268,11 +268,44 @@
         </v-col>
         <v-col
           class="text-left"
+          style="align-self:center!important"
           cols="9"
         >
-          <span style="color:#BDBDBD!important; font-size:90%; line-height:80%; align-items:center">The material contained on this website is based upon work supported by NASA under award No. 80NSSC21M0002. Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Aeronautics and Space Administration.</span>
+          <div style="color:#BDBDBD!important; font-size:70%; line-height:0.8em; align-self:center!important">
+            The material contained on this website is based upon work supported by NASA under award No. 80NSSC21M0002. Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Aeronautics and Space Administration.
+          </div>
         </v-col>
       </v-row>
+      <v-snackbar
+        v-model="show_snackbar"
+        timeout="200000"
+        transition="fab-transition"
+        top
+        center
+        color="info"
```

Comment:
I think this could just be the default color rather than the guideline color (so could remove `color="info"`